include/omath/pathfinding/NavigationMesh.hpp

```diff
@@ -8,7 +8,7 @@
 #include <expected>
 #include <vector>
 #include <string>
-
+#include "../omath_export.h"
```

Comment:
include path must be absolute "omath/omath_export.hpp", also all omath header files must be with .hpp extension not .h

Reply:
Done.